tests/Integration/Behaviour/Features/Context/Domain/AttributeFeatureContext.php

```diff
@@ -64,4 +64,56 @@ public function assertNamedAttributeExists(string $attributeReference, string $n
         Assert::assertNotNull($foundAttributeId, sprintf('Attribute named "%s" was not found', $name));
         $this->getSharedStorage()->set($attributeReference, $foundAttributeId);
     }
+
+    /**
+     * @Given I associate attribute ":attributeReference" with shops ":shopReferences"
+     *
+     * @param string $attributeReference
+     * @param string $shopReferences
+     *
+     * @return void
+     */
+    public function associateAttributeWithShops(string $attributeReference, string $shopReferences): void
```

Comment:
Since these steps are based purely on legacy code and no CQRS commands, they should be in the upper namespace `tests/Integration/Behaviour/Features/Context/AttributeFeatureContext.php` (same for the `assertAttributeIsNotAssociatedToShops` end point)